services/docker/docker-size.spec.js

```diff
@@ -3,40 +3,105 @@ import DockerSize from './docker-size.service.js'
 import { sizeDataNoTagSemVerSort } from './docker-fixtures.js'
 
 describe('DockerSize', function () {
-  test(DockerSize.prototype.transform, () => {
-    given({
-      tag: '',
-      sort: 'date',
-      data: { results: [{ name: 'next', full_size: 219939484 }] },
-    }).expect({
+  test(DockerSize.prototype.noTagWithDateSortTransform, () => {
+    given(
+      {
+        count: 0,
+        results: [
+          {
+            full_size: 219939484,
+            name: 'next',
+            images: [{ architecture: 'amd64', size: 219939484 }],
+          },
+        ],
+      },
+      'amd64'
+    ).expectError('Not Found: repository not found')
+    given(
+      {
+        count: 152,
+        results: [
+          {
+            full_size: 300000000,
+            name: 'next',
+            images: [{ architecture: 'amd64', size: 219939484 }],
+          },
+        ],
```

Comment:
These tests in this file are a bit confusing to read because the test data isn't logically consistent. Can we make the `count` values match the number of items in the array i.e: if there's 1 thing in the array, count should be 1 (not 152), if we're relying on matching `count === 0` in the test data, the array should actually have 0 things in it (not 1), etc.

Reply:
You're right; updated